### ISSUE TITLE
Dev Server 2.8 - Live bundle javascript fails to load

### ISSUE THREAD

**Issue by mtraynham** [webpack/webpack-dev-server#1080]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Live bundle IFrame fails to load, Chrome shows:
```js
Uncaught TypeError: e.indexOf is not a function
    at ge.fn.init.ge.fn.load (live.bundle.js:24)
    at HTMLDocument.r (live.bundle.js:1)
    at u (live.bundle.js:24)
    at l (live.bundle.js:24)
```

**If the current behavior is a bug, please provide the steps to reproduce.**
Run the IFrame live bundle

**What is the expected behavior?**
It runs correctly.  Looks like you guys refactored the client heavily with #1058.  The bug is here, https://github.com/webpack/webpack-dev-server/blob/master/client/live.js#L90. `iframe.load(cb)` should probably be `iframe.on('load', cb)`.

**Please mention your webpack and Operating System version.**
Linux Mint 18.3
Webpack Dev Server 2.8
Webpack 3.5

**Comment by shellscape** [comment]
@mtraynham there actually wasn't a big refactor, it was a linting implementation and cleanup. looking at the original code, https://github.com/webpack/webpack-dev-server/pull/1058/files#diff-7ca921e48f080bb70b26956b90f5229bL86, `iframe.load` has existed for some time. 

I'm also not sure what "Run the IFrame live bundle" is referring to. Are you speaking of the `modus-iframe` example?

**Comment by mtraynham** [comment]
Kind of looked like it was calling this [api.jquery.com/load](https://api.jquery.com/load/) function instead.

**Comment by shellscape** [comment]
(going off of the `modus-live` example) Yeah I think you're right. What's odd is that it's a valid jQuery object and element array, but the `load` method isn't proxying to the event handler. I reverted the code and the same symptoms exist. Looking at the PR, we updated some deps as well:

```diff
-    "jquery": "^2.2.0",
+    "jquery": "^3.2.1",
```

So that may have had something to do with it. I tested using `.on` instead and it did indeed resolve the issue. I'll prepare a fix and we'll get that out today as 2.8.1.

Thanks for triaging the issue.
